app/controllers/admin/claims_controller.rb

```diff
@@ -15,7 +15,7 @@ def show
   end
 
   def payroll
-    claims = Claim.includes(eligibility: [:claim_school, :current_school]).approved.order(:approved_at)
+    claims = Claim.preload(eligibility: [:claim_school, :current_school]).joins(:check).approved.order("checks.created_at")
```

Comment:
```suggestion
    claims = Claim.preload(eligibility: [:claim_school, :current_school]).approved.order("checks.created_at")
```

The join is already in `approved`.